### PR TITLE
Final round of docs reading and contributions

### PR DESCRIPTION
**Pull request by olberger**
I've now almost read all of the new docs, and here are a few more contributions ;)

### CODE REVIEW

File: docs/hacking/index.rst [hunk 1]
```diff
@@ -1,6 +1,8 @@
 Hacking on Antidote
 ===================
 
+This section is meant for people interested in developping on Antidote.
```
Comment (Mierdin) [inline docs/hacking/index.rst]:
```suggestion
This section is meant for people interested in developing on Antidote.
```

File: docs/hacking/platform.rst [hunk 2]
```diff
@@ -33,6 +36,8 @@ Hacking on Syringe
     This section discusses technical steps for building Syringe yourself when working on the code. See
     :ref:`here <contrib-platform>` for instructions on how to contribute your changes back to the project.
 
+Syringe is mainly an application written in Go providing a Web API.
```
Comment (Mierdin) [inline docs/hacking/platform.rst]:
```suggestion
Syringe is mainly an application written in Go which provides orchestration for lesson resources on Kubernetes, while providing an API for the web front-end to consume.
```

File: docs/platform/curricula/lessons/presentations.rst [hunk 3]
```diff
@@ -87,9 +87,10 @@ field that you should be aware of.
 SSH
   **Usage:** ``type: ssh``
 
-    This one is pretty straightforward. A very common use case for showing Endpoints is to provide
-    an interactive CLI terminal in the Antidote front-end. This is accomplished by connecting directly
-    to the Endpoint via SSH, and using that connection to provide the experience to the web.
+    This one is pretty straightforward. A very common use case for interacting with Endpoints is to provide
+    an interactive CLI (Command Line Interface) terminal in the Antidote front-end. This is accomplished by connecting directly
+    to the Endpoint via SSH, and using that connection to provide the experience to the web. Under the hood,
+    the front-end uses Guacamole's SSH tunnel and Web console.
```
Comment (Mierdin) [inline docs/platform/curricula/lessons/presentations.rst]:
```suggestion
    the front-end uses Guacamole's SSH tunneling capabilities.
```

No web console, we stripped all that out.

**Comment by Mierdin** [comment]
@olberger I'm back in the office now, any chance to look at my comments above?

**Comment by olberger** [comment]
Glad you're back with full thrust ;-) Go ahead !

**Pull request merged**